File: dayfour/javascript/RangeArray.js

function RangeArray(arr ){
    arr = [3,4,5,6,7,8,9]
    const largest = arr[0];
    for (let i = 0; i <arr.length-1; i++){
       if(arr[i]>arr[0]){
            largest == arr[i];
            }
        }
    
     let smallest = arr[0];
    for (let i = 0; i <arr.length-1; i++){
       if(arr[i] < arr[0]){
           smallest == arr[i];
            }
        }
    let  range = [largest - smallest + 1]
    
    for (let index =0; index<range.length; index++){
        range [index] = smallest;
        smallest = smallest+ 1;

    }
    return range;

File: dayfour/python/RangeArray.py
def RangeArray():
    largest = arr[0];
    for i in range(i, arr.length-1):
       if(arr[i]>arr[0]):
            largest = arr[i]
    
    smallest = arr[0];
    for i in range(1, arr.length-1):
       if(arr[i] < arr[0]):
           smallest = arr[i]
         
    range =  [largest - smallest + 1]
    
    for i in range(1, arr.length-1):
        range[index] = smallest;
        smallest += 1;
    return range;

